src/async_impl/request.rs

```diff
@@ -291,6 +292,7 @@ impl RequestBuilder {
     ///
     /// Serialization can fail if `T`'s implementation of `Serialize` decides to
     /// fail, or if `T` contains a map with non-string keys.
+    #[cfg(feature = "json")]
```

Comment:
We should probably include in the docs that this (and the other methods) requires the `json` feature enabled.